streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -97,7 +97,15 @@
      */
     public static final String CONSUMER_PREFIX = "consumer.";
 
-    // Prefix used to isolate producer configs from consumer configs.
+
+    /**
+     * Prefix used to provide default Topic configs to be applied when creating internal topics.
+     * These should be valid properties from {@link org.apache.kafka.common.config.TopicConfig}
```

Comment:
Nit `{@link org.apache.kafka.common.config.TopicConfig TopicConfig}`
Plus `.` at the end.